src/databases/VTK/VTKCommonPluginInfo.C

```diff
@@ -53,6 +53,9 @@ VTKCommonPluginInfo::GetDatabaseType()
 //    are present in the 1st file.
 //
 // ****************************************************************************
+
+#include <avtSTSDFileFormatInterface.h>
```

Comment:
Is there a particular reason that the STSD format header was moved to the code file but not the STMD format header?

Reply:
VTK reader is both STMD and STSD. Code gen tools don't allow (currently) for specifying both. STMD is specified in the .xml file, so its include is automatically added by the codegen. I added STSD to the .code file (with the definition of the method that needed it) so it would be included by the codegen.  It may be possible to add the include to the .xml file instead. It would appear with the rest of the includes at the top of the file then.  I will investigate this.

Reply:
Just tried it. Evidently xml2info doesn't support 'Include' specified in the .xml file (xml2atts does).